k8s/dialer.go

```diff
@@ -143,6 +145,12 @@ func (c *contextDialer) startDialerPod(ctx context.Context) (err error) {
 					Stdin:     true,
 					StdinOnce: true,
 					Args:      []string{"-u", "-", "OPEN:/dev/null,append"},
+					SecurityContext: &coreV1.SecurityContext{
+						Privileged:               new(bool),
+						AllowPrivilegeEscalation: new(bool),
```

Comment:
why do you need to set both to `false` ? Isn't this the default value anyway ?

Reply:
by default it's null, IIRC it caused troubles

Reply:
It should be explicitly ptr to false.

Reply:
I would too assumed it should work with zero/null value, but it doesn't.
It also applies to `RunAsNonRoot` despite the fact the user of the image is not `root`.